tests/test_data_processor.py

```
@@ -0,0 +1,42 @@
+import pytest
```

Ruff

`pytest` imported but unused

Remove unused import: `pytest`

(F401)